lib/rucio/web/rest/flaskapi/v1/auth.py

```diff
@@ -697,7 +697,7 @@ def get(self) -> 'ResponseReturnValue':
             # response.set_cookie('rucio-auth-token-created-at', value=str(time.time()))
             return response
         else:
-            return '', 400, headers
+            return generate_http_error_flask(status_code=400, exc="BadRequest", exc_msg="", headers=headers)
```

Comment:
```suggestion
            return generate_http_error_flask(status_code=400, exc=InvalidRequest.__name__, exc_msg="", headers=headers)
```

I would use `InvalidRequest` here, in line with `CodeOIDC.get()`. Please add an import to import `InvalidRequest` from `rucio.common.exception` as well.